nnpdf_data/nnpdf_data/filter_utils/star/jet_dijet_2009_200gev.py

```diff
@@ -0,0 +1,319 @@
+    ]
+
+    for fname in fnames:
+        if "3" in fname:
+            dfi = pd.read_csv(fname)
+            if "pT" in fname:
+                df3["pT"] = dfi["Parton Jet $p_T$ [GeV/c]"]
+
+            elif "ALL" in fname:
+                df3["ALL"] = dfi["$A_{LL}$"]
+                df3["stat_min"] = dfi["stat +"]
+                df3["stat_max"] = dfi["stat -"]
+                df3["sys_min"] = dfi["sys +"]
+                df3["sys_max"] = dfi["sys -"]
+
+            df3["abs_eta_min"] = 0.0
+            df3["abs_eta_max"] = 0.5
+            df3["abs_eta"] = (df3["abs_eta_min"] + df3["abs_eta_max"]) / 2
+            df3["sqrts"] = 200
+
+        elif "4" in fname:
+            dfi = pd.read_csv(fname)
+            if "pT" in fname:
+                df4["pT"] = dfi["Parton Jet $p_T$ [GeV/c]"]
+
+            if "ALL" in fname:
+                df4["ALL"] = dfi["$A_{LL}$"]
+                df4["stat_min"] = dfi["stat +"]
+                df4["stat_max"] = dfi["stat -"]
+                df4["sys_min"] = dfi["sys +"]
+                df4["sys_max"] = dfi["sys -"]
+
+            df4["abs_eta_min"] = 0.5
+            df4["abs_eta_max"] = 1.0
+            df4["sqrts"] = 200
+
+        elif "5" in fname:
+            dfc_col = pd.read_csv(fname)
+
+            dfc = pd.DataFrame()
+            biny = 1
+            for i in range(len(dfc_col)):
+                if dfc_col.loc[i, "binx"] == "-":
+                    biny = float(dfc_col.loc[i, "val"])
+                else:
+                    binx = float(dfc_col.loc[i, "binx"])
+                    dfc.loc[binx, biny] = dfc_col.loc[i, "val"]
+
+            dfc = dfc.astype("float")
+
+        else:
+            print("ERROR: Unknown table number detected! Check input files.")
+
+    df = pd.concat([df3, df4], ignore_index=True)
+    df["pol"] = POL_UNC * df["ALL"]
+    df["lumi_ue"] = LUMI_UNC
+
+    for i in range(len(df)):
+        df.loc[i, "stat"] = df.loc[i, "stat_max"]
+        df.loc[i, "sys"] = df.loc[i, "sys_max"]
+    return df, dfc
+
+
+def read_2jet_data(topology):
+    fname = f"../../new_commondata/STAR_2009_2JET_{topology}_200GEV/rawdata/Table.yaml"
+    df = pd.DataFrame()
+    with open(fname, "r") as file:
+        data = yaml.safe_load(file)
+
+    if topology in ["A", "B", "C"]:
+        Msub = data["independent_variables"][0]["values"]
+        Gsub = data["dependent_variables"][0]["values"]
+
+        for i in range(len(Msub)):
+
+            df = pd.concat(
+                [
+                    df,
+                    pd.DataFrame(
+                        {
+                            "m_low": [Msub[i]["low"]],
+                            "m_high": [Msub[i]["high"]],
+                            "ALL": [Gsub[i]["value"]],
+                            "stat": [Gsub[i]["errors"][0]["symerror"]],
+                            "sys": [Gsub[i]["errors"][1]["symerror"]],
+                            "pol": [POL_UNC * Gsub[i]["value"]],
```

Comment:
Here we should drop both pol and lumi, because are already included in the systematics, accordying
to the paper:

```
The systematic uncertainty on ALL includes the beam polarization uncertainty, the relative luminosity uncertainty, the underlying event systematic uncertainty, the trigger and reconstruction bias uncertainty, and the residual transverse polarization uncertainty
```